src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx

```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
+
+To change a condition, go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert Conditions**</DNT> and click a condition.
+
+* To [change the condition's name](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions#rename-condition) select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list.
+* To update the values for the required Critical (red) or optional Warning (yellow) [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert), select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list..
+* To update the condition's [runbook URL](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/provide-runbook-instructions-alerts), select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list.
+
+## Enable/disable a condition [#disable-conditions]
+
+You can enable <img title="Alerts v3: Condition enabled" alt="041715icon-condition-on" src={accountsConditionOn}/> or disable <img title="Alerts v3: Condition disabled" alt="041715icon-condition-off" src={accountsConditionOff}/> any conditions, and the policy will continue to apply. To disable or re-enable a condition:
+
+1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert Conditions**</DNT>. Then, from the list of <DNT>**Alert conditions**</DNT>, use the toggle to enable or disable the condition.
+2. Click the <DNT>**On/Off**</DNT> switch to toggle it.
+
+If you [copy a condition](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/copy-alert-conditions), it automatically saves it in the new policy as disabled (<DNT>**Off**</DNT>), even if the condition was enabled (<DNT>**On**</DNT>) in the original policy.
+
+## Delete a condition [#delete-conditions]
+
+If a policy has multiple conditions, you can delete any number of them, and the remaining conditions for the policy will continue to apply. To turn a condition off but keep it with the policy, [disable](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/disable-or-delete-alert-policies-conditions#condition-on-off) it. To delete one or more conditions:
+
+1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert Conditions**</DNT>.
+2. From the list of <DNT>**Alert conditions**</DNT>, select a condition, then click <DNT>**Delete**</DNT> from the ellipses menu (...).
+
+    <Callout variant="tip">
+      If you don't see the delete button, your account admin may have disabled condition deletion for your organization.
+    </Callout>
+
+## Troubleshoot the alert condition page [#troubleshoot]
 
-If you see an empty signal in the history chart, consider one of the following:
+If you see an empty signal in the history chart on the **Alert Conditions** page, consider one of the following:
```

Comment:
This should be consistent with the title.  Singular `condition`.

Reply:
Fixed!